Doc/library/sys.rst

```diff
@@ -1122,6 +1122,9 @@ always available.
    ``cache_tag`` is set to ``None``, it indicates that module caching should
    be disabled.
 
+   *pyc_magic_number* is the magic number used to identify that cached ``.pyc``
```

Comment:
Can we add the ``versionadded`` directive here?

Reply:
Obsoleted, since we ended up [making this private](https://github.com/python/cpython/pull/122243#discussion_r1692090610)